src/sentry/grouping/enhancement-configs/common@2019-03-23.txt

```diff
@@ -62,3 +62,11 @@ family:native function:"?[[]SentryClient *"                       -app -group
 family:native function:"?[[]SentryHub *"                          -app -group
 family:native function:"?[[]RNSentry *"                           -app -group
 family:native function:"?[[]SentrySDK *"                          -app -group
+
+# Java
+# Match $$EnhancerByGuice$$ and $$FastClassByGuice$$ with a single rule:
+family:java module:*ByGuice$$*                                       -app -group
```

Comment:
Would be nice to have rules for spring CGLIB as well:
$$EnhancerBySpringCGLIB$$ and $$FastClassBySpringCGLIB$$